caffe2/python/data_parallel_model_test.py

```diff
@@ -102,6 +102,7 @@ def test_equiv(self):
             result_8gpus = self.run_model(range(8))
             self.assertTrue(np.allclose(result_1gpus, result_8gpus))
 
+    @unittest.skipIf(workspace.NumCudaDevices() < 4, "Need at least 4 GPUs.")
     def test_checkpoint_params(self):
         def add_input_ops(model):
             pass
```

Comment:
Yeah, there's an error about not finding a device with ordinal 2. I'll repro later and paste the error.